playwright-e2e/tests/dsm/osteo/os2-display-verification.spec.ts

```diff
@@ -0,0 +1,889 @@
+      await savedFilterSection.openPanel();
+      await savedFilterSection.exists(filterNewYorkResidence);
+      await savedFilterSection.exists(filterDidNotConsentToTissue);
+      await savedFilterSection.exists(filterReturnOfResults);
+
+      await savedFilterSection.delete(filterNewYorkResidence); //Will move these to a test occurring later - putting here to not clog saved filter before tests are finalized
+      await savedFilterSection.delete(filterDidNotConsentToTissue);
+      await savedFilterSection.delete(filterReturnOfResults);
+    })
+  })
+
+  test(`${StudyName.OSTEO2}: Verify general participant list webelements`, async ({ page, request }) => {
+    test.slow();
+    navigation = new Navigation(page, request);
+    await new Select(page, { label: 'Select study' }).selectOption(StudyName.OSTEO2);
+
+    const participantListPage = await navigation.selectFromStudy<ParticipantListPage>(Study.PARTICIPANT_LIST);
+    await participantListPage.waitForReady();
+
+    const participantListTable = participantListPage.participantListTable;
+    await participantListTable.assertDisplayedHeaders({ checkDefaultFilterOfStudy: true, studyName: StudyName.OSTEO2 }); //check to make sure test is starting from clean slate
+
+    await test.step(`Verify that the Select All option is displayed`, async () => {
+      await participantListPage.assertSelectAllDisplayed();
+    })
+
+    await test.step(`Verify the ${StudyName.OSTEO2} quick filters are displayed`, async () => {
+      const quickFilters = participantListPage.quickFilters;
+      await quickFilters.assertQuickFilterDisplayed(QuickFilter.MEDICAL_RECORDS_NOT_REQUESTED_YET);
+      await quickFilters.assertQuickFilterDisplayed(QuickFilter.MEDICAL_RECORDS_NOT_RECEIVED_YET);
+      await quickFilters.assertQuickFilterDisplayed(QuickFilter.TISSUE_NEEDS_REVIEW);
+      await quickFilters.assertQuickFilterDisplayed(QuickFilter.TISSUE_NOT_REQUESTED_YET);
+      await quickFilters.assertQuickFilterDisplayed(QuickFilter.TISSUE_REQUESTED_NOT_RECEIVED_YET);
+      await quickFilters.assertQuickFilterDisplayed(QuickFilter.PARTICIPANT_WITHDRAWN);
+      await quickFilters.assertQuickFilterDisplayed(QuickFilter.AOM_IN_NEXT_SIX_MONTHS);
+      await quickFilters.assertQuickFilterDisplayed(QuickFilter.AOM_IN_LAST_SIX_MONTHS);
+      await quickFilters.assertQuickFilterDisplayed(QuickFilter.LOST_TO_FOLLOW_UP_AOM_WAS_OVER_ONE_MONTH_AGO);
+      await quickFilters.assertQuickFilterDisplayed(QuickFilter.UNDER_AGE);
+      await quickFilters.assertQuickFilterDisplayed(QuickFilter.PHI_REPORT);
+    })
+
+    await test.step(`Verify that the Download List and Bulk Cohort Tagis displayed`, async () => {
+      await participantListPage.assertDownloadListDisplayed();
+      await participantListPage.assertBulkCohortTagDisplayed();
+    })
+
+    const customizeViewPanel = participantListPage.filters.customizeViewPanel;
+    await customizeViewPanel.open();
+
```

Comment:
Checking all customize view options looks long - but only seems to take the test ~ 1 min, 50 seconds to check